src/main/java/org/myrobotlab/service/Tracking.java

```diff
@@ -303,6 +305,66 @@ public ServiceConfig getConfig() {
     return config;
   }
 
+  static public Map<String, ServiceConfig> getDefaultConfig() {
+
+    Map<String, ServiceConfig> config = new HashMap<>();
+
+    RuntimeConfig runtime = new RuntimeConfig();

```

Comment:
i thought we were going to get rid of runtime config here because we are using a linked hash map for the name/service config pairs ..